core/src/main/java/org/arquillian/smart/testing/hub/storage/local/TemporaryInternalFiles.java

```diff
@@ -12,4 +15,19 @@ public static String getScmChangesFileName(){
     public static String getJunit5PlatformVersionFileName(String pluginArtifactId){
         return pluginArtifactId + "_" + JUNIT_5_PLATFORM_VERSION;
     }
+
+    public String getTestReportDirectoryName(){
+        return TEMP_REPORT_DIR;
+    }
+
+    public LocalStorageDirectoryAction getTestReportDirectoryAction(String rootDir){
+        return getTestReportDirectoryAction(new File(rootDir));
+    }
+
+    public LocalStorageDirectoryAction getTestReportDirectoryAction(File rootDir){
```

Comment:
Minor thing really, but I'm wondering if we should stick to `get` from Java Bean convention or rather have a hint that action is created, so `createTestReportDirectoryAction` instead.

Reply:
+1, I was thinking about the same - `createTestReportDirectoryAction` sounds more natural for me. I just kept the method naming in the same way as it is used in other cases in this class. But looking at it this morning, the other methods just return the constants, but this one creates the class. So I'll change it